src/infrastructure/markup/rule/PhabricatorBugzillaLinkRule.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+final class PhabricatorBugzillaLinkRule extends PhutilRemarkupRule {
+
+  public function getPriority() {
+    return 360.0;
```

Comment:
I'm not sure what the best value here is.
This is copied from `PhabricatorYoutubeRemarkupRule.php` and added by 10, for not much specific reason.